nanopore-geometry-experiment.package/conf/flowir_package.yaml

```diff
@@ -266,6 +266,7 @@ output:
 
 platforms:
```

Comment:
Now that the Python scripts were moved from `bin/` to `reference-scripts/`, one might need to redefine the `PATH` environment variable otherwise it will not find the new location.

Reply:
Don't worry about the delay! I'm just hoping to use your workflow in the upcoming docs for ST4SD so I'm proposing a couple of changes to demonstrate best practices.

As for the reference scripts I was thinking about completely removing them but I left them in as a reference. The issue is that the components reference the images that exists in their respective images instead of the ones in the workflow project.

Here's an example: https://github.com/st4sd/nanopore-geometry-experiment/blob/85244f5c84d6c0e02aa7a2e1685b004cd0b3cdd2/conf/flowir_package.yaml#L186

Because there's no `/` in the executable it gets resolved using $PATH which cannot include `$INSTANCE_DIR/bin`. The way to use the bundled scripts would be to write `executable: bin/zeo.sh`. That would also imply that there needs to be an `__init__.py` file under `bin/modules` so that the script `geo_amalgamate.py` can import `modules.parse_zeopp.py`

In the end I thought that users might want to look at the contents of scripts which is why I moved them to a `reference-scripts` dir so that its clear that they're not actually used.